Automattic/jetpack
# Facebook widget: hide-cta and small-header

#### Changes proposed in this Pull Request:
Fixes #9270


* Fixes this issue: https://github.com/Automattic/jetpack/issues/9270
* Adds an option to hide call to action button
* Adds an option to show small header instead of big one
* By default these two options will be off

#### Testing instructions:






* Go to widgets page, add Facebook widget, add URL and save. 
* Go to front page, inspect the facebook widget, you will see `data-hide-cta="false" data-small-header="false"`
* Go back to widget settings and enable both `hide call to action` and `show small header` options.
* Now you will see  `data-hide-cta="true" data-small-header="true"` in the element inspector.

#### Proposed changelog entry for your changes:

* Facebook widget now supports all new settings for the [Facebook page plugin](https://developers.facebook.com/docs/plugins/page-plugin/)

#### Screenshot:

![Screen Shot on 2019-10-14 at 08:00:19](https://user-images.githubusercontent.com/20779676/66726726-b142f980-ee58-11e9-828c-a45688349fa5.png)

#### Other notes:

The third functionality `adapt-container-width` is covered by this https://github.com/Automattic/jetpack/issues/9889 issue and this https://github.com/Automattic/jetpack/pull/13740 PR.
